test/test_string_map.cpp

```diff
@@ -115,10 +140,6 @@ TEST(test_string_map, getters) {
 
   // null for capacity/size
   {
-    rcutils_string_map_t string_map = rcutils_get_zero_initialized_string_map();
-    ret = rcutils_string_map_init(&string_map, 0, allocator);
```

Comment:
How are you able to remove the init here? Presumably it was done for a reason.

Reply:
Oops - reverted

Reply:
Now, the initialization is done in the fixture so this is not necessary anymore here.

Reply:
oops - I reverted the change, `getters_capacity_null_capacity` and `getters_size_null_size` are operating on an initialized list again.